client/src/pages/LoginPage.js

```diff
@@ -44,7 +61,7 @@ export default function() {
             <Input type="email" placeholder="Email" fullWidth/>
             <Input type="password" placeholder="Password" fullWidth/>
             <a className="subText" href="#">Forgot your password?</a>
-            <Button>{text}</Button>
+            <Button to="/home">{text}</Button>
```

Comment:
we are going to route this somewhere else right?

Reply:
Hmm it depends on however we setup the spotify auth, you can attach it to a function that sends a request to `/spotifyauth` or something. but tentatively I have it set up to route to /home, in which /home checks for tokens etc.